src/resources-view_impl.cxx

```diff
@@ -22,8 +22,100 @@ ResourcesViewImpl::ResourcesViewImpl (
   {
     parent->remove (*m_top_widget);
   }
+
+  Gtk::ToolButton* btn;
+  builder->get_widget ("tb-add", btn);
+  btn->signal_clicked ().connect (
+    sigc::mem_fun (*this, &ResourcesViewImpl::cb_on_add_clicked));
+
+  builder->get_widget ("tb-cut", btn);
+  btn->signal_clicked ().connect (
+    sigc::mem_fun (*this, &ResourcesViewImpl::cb_on_cut_clicked));
+
+  builder->get_widget ("tb-copy", btn);
+  btn->signal_clicked ().connect (
+    sigc::mem_fun (*this, &ResourcesViewImpl::cb_on_copy_clicked));
+
+  builder->get_widget ("tb-paste", btn);
+  btn->signal_clicked ().connect (
+    sigc::mem_fun (*this, &ResourcesViewImpl::cb_on_paste_clicked));
+
+  builder->get_widget ("tb-delete", btn);
+  btn->signal_clicked ().connect (
+    sigc::mem_fun (*this, &ResourcesViewImpl::cb_on_delete_clicked));
+
+  builder->get_widget ("tb-indent", btn);
+  btn->signal_clicked ().connect (
+    sigc::mem_fun (*this, &ResourcesViewImpl::cb_on_indent_clicked));
+
+  builder->get_widget ("tb-unindent", btn);
```

Comment:
I don't think indent and unindent is necessary in resources.

Reply:
+1. 

Resources are only one level. We could use a liststore.